charts/shoot-core/components/charts/network-policies/templates/node-exporter-from-vpn.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
```

Comment:
I just had a view and there is already another policy https://github.com/mvladev/gardener/blob/4dab9070a5ece73b2a6ad1a585a036c4a410108b/charts/shoot-core/components/charts/network-policies/templates/allow-from-seed.yaml#L1-L24

which allows for Pods to be labeled with `networking.gardener.cloud/from-seed: allowed`.
So this NetworkPolicy would not be needed and you can simply label the node-exporter with `networking.gardener.cloud/from-seed: allowed`

Reply:
thanks! i've modified per comments